### ISSUE TITLE
script error.

### ISSUE THREAD

**Issue by brucesa85** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#172]
your script is missing Open and Close brackets { }
Fixed Lines 19 and 22
<#
.SYNOPSIS
This script allows you to uninstall the Microsoft Teams app and remove Teams directory for a user.
.DESCRIPTION
Use this script to clear the installed Microsoft Teams application. Run this PowerShell script for each user profile for which the Teams App was installed on a machine. After the PowerShell has executed on all user profiles, Teams can be redeployed.
#>

$TeamsPath = [System.IO.Path]::Combine($env:LOCALAPPDATA, 'Microsoft', 'Teams')
$TeamsUpdateExePath = [System.IO.Path]::Combine($env:LOCALAPPDATA, 'Microsoft', 'Teams', 'Update.exe')

try
{
    if (Test-Path -Path $TeamsUpdateExePath) {
        Write-Host "Uninstalling Teams process"

        # Uninstall app
        $proc = Start-Process -FilePath $TeamsUpdateExePath -ArgumentList "-uninstall -s" -PassThru
        $proc.WaitForExit()
    }{
    Write-Host "Deleting Teams directory"
    Remove-Item –Path $TeamsPath -Recurse
	}
}
catch
{
    Write-Error -ErrorRecord $_
    exit /b 1
}

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c642df17-1656-7c29-9fc3-04f70689f902
* Version Independent ID: 63bef712-cfb2-9efc-0b62-f68b0968b84c
* Content: [PowerShell Script Sample - Assists with Microsoft Teams Deployment Clean Up](https://docs.microsoft.com/en-us/microsoftteams/scripts/powershell-script-teams-deployment-clean-up)
* Content Source: [Teams/scripts/Powershell-script-teams-deployment-clean-up.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/scripts/Powershell-script-teams-deployment-clean-up.md)
* Service: **msteams**
* GitHub Login: @ninadara
* Microsoft Alias: **ninadara**

**Comment by kenwith** [comment]
@brucesa85 - would you mind submitting a Pull Request and I will get @ninadara to review and we can get it merged in?

Thanks for submitting!

**Comment by kenwith** [comment]
@tonysmit - do you know who owns this script? I have been emailing with Bruce and we might need to add {} to make the script run all of the time without errors but I am not SME on this and not sure who we can get to validate.

**Comment by tonysmit** [comment]
@juturu or @voltarone - can you help us with this?

**Comment by kenwith** [comment]
@juturu and @voltarone - Bruce noticed that sometimes the script worked with the curly braces and sometimes without. Any idea what might cause this? If it didn't work with the curly braces he took them out and it would work and if it didn't work without the curly braces he put them in and it worked. We are thinking something in the environment but not sure and looking for guidance. Thanks!

**Comment by kenwith** [comment]
Adding in the curly braces and closing this one out. If anything else please re-open. Thanks for this!

**Comment by MelQL** [comment]
Those {} added at 19 and 22 by brucesa85 actually cause the script to not fully delete the Teams folder in the user's appdata\local\microsoft\teams for me.  If I have those 2 curly braces, .dead and update.exe are left behind.  If I remove the curly braces at 19 and 22, the folder gets fully deleted.  I don't actually understand why- it seems like the script should work either way.   

I tried multiple times both ways (maybe 10 times each) and it always deleted everything if I did not have the braces and always failed to delete everything if I had the braces.

**Comment by LolaJacobsen** [comment]
@ninadara Could you take a look at this? Looks like we need a fix to the script in Powershell-script-teams-deployment-clean-up.md.

**Comment by LolaJacobsen** [comment]
Per Nishanth, this script was written by Hari Krishna Reddy Juturu. I'll assign it to him for feedback.

**Comment by brucesa85** [comment]
There is something up with the script. Sometimes it works with the curly{} braces sometimes it doesn't. I actually gave up on it for now as I couldn't get it to work 100% with or without the braces :/

**Comment by StumpyRob** [comment]
Having the same  issue, It seems s though there is something wrong with the brackets on the exception handling of try and catch I commented the section out to troubleshoot but it seems as though there is another issue with the TeamsPath variable, it looks as though it is passing: â€“ at the beginning of the string causing an error when "Remove-Item" is ran. 

Not sure where the -  â€“ - is coming from though...

**Comment by StumpyRob** [comment]
Update: looks as though the â€“ is a special character that has been improperly encoded for utf-8

**Comment by kenwith** [comment]
@juturu - can you help here?

**Comment by AndreaBarr** [comment]
@brucesa85 Thank you for this feedback

We need to track down the owner of this script so I am adding it to our backlog to the Teams/Skype for Buisness team so we can get this handled appropriatly. 

I was able to get this into the backlog for the Teams/Skype for Business team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
